#! /bin/sh /usr/share/dpatch/dpatch-run
## 0015-upstream-fix-logic-error-in-imapoptions.dpatch by Sven Mueller <debian@incase.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: fix a small logical mistake in imapoptions description 

@DPATCH@
--- a/lib/imapoptions
+++ b/lib/imapoptions
@@ -42,7 +42,7 @@
 .\" AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING
 .\" OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 .\" 
-.\" $Id: imapoptions,v 1.36 2006/03/30 15:49:58 murch Exp $
+.\" $Id: imapoptions,v 1.37 2006/06/05 23:31:22 murch Exp $
 .SH NAME
 imapd.conf \- IMAP configuration file
 .SH DESCRIPTION
@@ -801,7 +801,7 @@ are listed with ``<none>''.
 /* The cyrusdb backend to use for the subscriptions list. */
 
 { "syslog_prefix", NULL, STRING }
-/* String to be appended to the process name in syslog entries. */
+/* String to be prepended to the process name in syslog entries. */
 
 { "temp_path", "/tmp", STRING }
 /* The pathname to store temporary files in */
